fdbserver/QuietDatabase.actor.cpp

```diff
@@ -731,13 +731,15 @@ ACTOR Future<Void> repairDeadDatacenter(Database cx,
 			    .detail("PrimaryDead", primaryDead);
 			g_simulator->usableRegions = 1;
 
-			state std::vector<AddressExclusion> servers = g_simulator->getAllAddressesInDCToExclude(
-			    primaryDead ? g_simulator->primaryDcId : g_simulator->remoteDcId);
-			wait(excludeServers(cx, servers, false));
-			TraceEvent(SevWarnAlways, "DisablingFearlessConfiguration")
-			    .detail("Location", context)
-			    .detail("Stage", "ServerExcluded")
-			    .detail("Servers", describe(servers));
+			if (remoteDead) {
+				state std::vector<AddressExclusion> servers =
+				    g_simulator->getAllAddressesInDCToExclude(g_simulator->remoteDcId);
+				wait(excludeServers(cx, servers, false));
+				TraceEvent(SevWarnAlways, "DisablingFearlessConfiguration")
```

Comment:
Move this TraceEvent before `excludeServers`, like other stages?

Reply:
It's better to be after, because the missing of this event means the `excludeServers` call hasn't returned. If it's before, we will see two consecutive `DisablingFearlessConfiguration` events, not giving much information.

Reply:
I think the previous `DisablingFearlessConfiguration` should be placed right before below `changeConfig`, since the `Stage` is `Repopulate`.

Also like recovery, we put the TraceEvent before the action so we know what it is currently stuck at. Feels like a convention in FDB code base.

Reply:
Sounds good. I've reordered these two events. PTAL!